src/util.ml

```diff
@@ -964,7 +964,8 @@ value wprint_geneweb_link conf href s =
 
 value reference conf base p s =
   if conf.cancel_links || is_hidden p then s
-  else "<a href=\"" ^ commd conf ^ acces conf base p ^ "\">" ^ s ^ "</a>"
+  else "<a href=\"" ^ commd conf ^ acces conf base p ^ "\" 
```

Comment:
Also, since you modified the code, maybe something like this would be better ?

```ocaml
else
  String.concat "" [""<a href=\"""; commd conf; ...]
```